build-tools/automation/azure-pipelines.yaml

```diff
@@ -685,6 +685,76 @@ stages:
         configuration: $(ApkTestConfiguration)
         artifactName: Test Results - APK Instrumentation - macOS
 
+    # Plots: Utilize command-line utility built here to send plot information contained in CSV files to AppInsights
+    - task: MSBuild@1
+      displayName: build plots-to-appinsights
+      inputs:
+        solution: build-tools/plots-to-appinsights/ProcessPlotCSVFile.csproj
+        configuration: $(ApkTestConfiguration)
+        msbuildArguments: >
+          /restore
+          /t:Build
+          /v:normal
+          /bl:$(System.DefaultWorkingDirectory)/bin/Tools$(ApkTestConfiguration)plots-to-appinsights.binlog
+      continueOnError: true
+      condition: and(succeeded(), or(eq(variables['Build.Reason'], 'IndividualCI'), eq(variables['Build.Reason'], 'Manual')))
```

Comment:
@mjbond-msft What are these additional `condition`'s for?

@jonpryor @pjcollins Should we run this on PR builds?  It feels like we shouldn't.  On one hand we wouldn't catch a regression before we merged, but we also don't want PR tests messing up our graphs.

Reply:
the conditions are there to ensure that we do not collect or report plot information during PR builds. basically, applied the same policy as found for the current `xamarin-android` Jenkins build where plots are only reported for CI builds

Reply:
i should note that i'm enabling plots for _manual_ builds. this mostly serves as a convenience to be able to launch a manual build against a feature branch to test that plots are getting processed correctly since they won't be processed as part of a PR

Reply:
@jpobst regarding the comment "don't want PR tests messing up our graphs" the Build.Reason (PR, CI, Manual) is included in the telemetry being sent. so _if_ the decision is made to include plots as part of PRs, the telemetry query backing the graph would simply filter on Build.Reason == "CI" (and i'll make sure i have this in the backing query) and so PR plots wouldn't mess up the graphs